### PR TITLE
Added support for CamelCase to RESTModelMixin. See #2480.

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

File: openslides/utils/utils.py [hunk 1]
```diff
@@ -1,6 +1,23 @@
+import re
 import roman
 
 
+first_re = re.compile('(.)([A-Z][a-z]+)')
+second_re = re.compile('([a-z0-9])([A-Z])')
```
Comment (ostcar) [inline openslides/utils/utils.py]: Maybe you could rename this two variables. Keep in mind, that we could add more regular expressions for other function in this file in the future.

File: openslides/utils/utils.py [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+
+
+def convert_camel_case(text):
```
Comment (ostcar) [inline openslides/utils/utils.py]: The name is unclear. When I read the code, I though a string was converted to CamelCase. For example a function
```python
covert_lower_case(my_string)
```
should make my_string lower case and not expect a lower_case_string.

Proposal: Rename the function to camel_to_snake_case.

File: openslides/utils/utils.py [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
+    """
+    Converts ThisText to this-text.
```
Comment (ostcar) [inline openslides/utils/utils.py]: What about "ThatText"? :)

File: openslides/utils/utils.py (hunk 1 above)
Comment (normanjaeckel) [inline openslides/utils/utils.py]: Changed.

File: openslides/utils/utils.py (hunk 2 above)
Comment (normanjaeckel) [inline openslides/utils/utils.py]: Changed.

File: openslides/utils/utils.py (hunk 3 above)
Comment (normanjaeckel) [inline openslides/utils/utils.py]: NoProblemWithThisText.

File: openslides/utils/utils.py [hunk 4]
```diff
@@ -1,6 +1,23 @@
...
+
+    Credits: epost.
+    See http://stackoverflow.com/questions/1175208
```
Comment (normanjaeckel) [inline openslides/utils/utils.py]: Done.

**Comment by normanjaeckel** [comment]
All done now. Please test this by checking out master and adding a projector message. Then check this branch and just make sure everything runs successfully (projector message, chat, projection defaults, motion blocks, motion change recommendation).

**Pull request merged**